rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -57,3 +69,39 @@ def test_decoders_initialization(evm_transaction_decoder):
         'convex',
         'sushiswap-v2',
     }
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045']])  # noqa: E501
+def test_no_logs_and_zero_eth(
+        database,
+        ethereum_manager,
+        function_scope_messages_aggregator,
+):
+    """Data taken from
+    https://etherscan.io/tx/0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f
+    """
```

Comment:
I belive this is the style @LefterisJP is following in the comments

```suggestion
    """
    Data taken from
    https://etherscan.io/tx/0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f
    """
```

Reply:
Yeah, alright. I just copy-pasted in from another decoder test